app/scicrunch_processing_common.py

```diff
@@ -32,6 +33,7 @@
     'inode/vnd.abi.scaffold+thumbnail': SCAFFOLD_THUMBNAIL,
     'inode/vnd.abi.scaffold.thumbnail+file': SCAFFOLD_THUMBNAIL,
     'inode/vnd.abi.scaffold.view+file': SCAFFOLD_VIEW_FILE,
+    'application/vnd.abi.context-information+json': CONTEXT_FILE,
```

Comment:
`application/x.vnd.abi.context-information+json` you are missing the initial 'x' here.

Reply:
ooops I thought it was going to match the others

Reply:
You probably need to pull in the latest changes from the upstream. Hugh has added a whole branched of the updated mimetypes few days ago.